Lib/test/test_largefile.py

```diff
@@ -45,6 +45,20 @@ def tearDownClass(cls):
             raise cls.failureException('File was not truncated by opening '
                                        'with mode "wb"')
 
+    @bigmemtest(size=_2G + 1024 * 1024, memuse=2)
+    def test_large_reads_writes(self, size):
+        # see issue #24658
+        requires('largefile',
+                 'test requires %s bytes and a long time to run' % size)
+        with self.open(TESTFN, "wb") as f:
+            b = b'x' * size
+            self.assertEqual(f.write(b), size)
+            self.assertEqual(f.tell(), size)
```

Comment:
Since this test is writing the full file, it will need to remove it as well so that the other tests will be set up as expected.

Reply:
Fixed indirectly, the test now reuses LargeFileTest.setUp().